src/constants.json

```diff
@@ -38,7 +38,8 @@
     "bidWon": "adUnitCode"
   },
   "ORDER": {
-    "RANDOM": "random"
+    "fixed": "fixed",
```

Comment:
now that we have language `const` support the value of a `constants` file feels diminished. Probably better to define these constant strings closer to where they are used in the code.